spec/src/main/asciidoc/jakarta-stl.adoc

```diff
@@ -1030,7 +1029,7 @@ content, then the action trims it and processes it further.
 
 | _escapeXml_ |
 _true_ | _boolean_
-|Deterrmines whether characters <,>,&,’,” in
+|Deterrmines whether characters <,>,&,'," in
```

Comment:
Minor type, not related to PR.

Reply:
I'll fix this typo in my next PR for minor corrections, I have a few to clean up that I've noticed while working through the doc. Good catch!

Reply:
Fixing here: https://github.com/eclipse-ee4j/jstl-api/pull/109